README.md

```diff
@@ -279,3 +279,23 @@ breaking change and would be subject to the announcement policy stated above.
 The `protoc-gen-go/generator` package exposes a plugin interface,
 which is used by the gRPC code generation. This interface is not
 supported and is subject to incompatible changes without notice.
+
+## Using protoc-gen-go in Continuous Integration (CI) Systems
+
+CI systems should produce repeatable builds.  Installing proto-gen-go via:
+
+```bash
+go get go get github.com/golang/protobuf/protoc-gen-go
+```
+
+within a CI system produces inherently non-reproducible builds, because 
```

Comment:
[nit] "reproducible" is a stronger property than repeatable, I'd think. (See this definition, for example, https://reproducible-builds.org/docs/definition/)